aws_xray_sdk/sdk_config.py

```diff
@@ -0,0 +1,35 @@
+import os
+
+
+class SDKConfig(object):
+    """
+    Global Configuration Class that defines SDK-level configuration properties.
+    It is recommended to only use the recorder to set this configuration's enabled
+    flag to maintain thread safety.
+    """
+    XRAY_ENABLED_KEY = 'AWS_XRAY_ENABLED'
+    __SDK_ENABLED = str(os.getenv(XRAY_ENABLED_KEY, 'true')).lower() != 'false'
+
+    @staticmethod
+    def sdk_enabled():
+        """
+        Returns whether the SDK is enabled or not.
+        """
+        return SDKConfig.__SDK_ENABLED
+
+    @staticmethod
+    def set_sdk_enabled(value):
```

Comment:
If an arbitrary value is passed, we should fallback to `true` and throw a warning says invalid configuration, rather than taking that arbitrary value and rely on python truthiness to determine. (5 means true while [] means false)

Reply:
That's a good point. Added fallback to true and explicit boolean checking.